props/saber_fett263_buttons.h

```diff
@@ -45,12 +85,44 @@ SaberFett263Buttons() : PropBase() {}
   bool swinging_ = false;
   void Loop() override {
     PropBase::Loop();
+    // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+    if(!SaberBase::IsOn()) {
+      if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
+        SaberBase::RequestMotion();
+      // Edit '250' value in line below to change swing on sensitivity, 250 ~ 400 work best in testing
+        if (!swinging_ && fusor.swing_speed() > 250) {
+          swinging_ = true;
+          Event(BUTTON_NONE, EVENT_SWING);
+        }
+        if (swinging_ && fusor.swing_speed() < 100) {
+          swinging_ = false;
+        }
+      }
+    } 
     if (!swinging_ && fusor.swing_speed() > 250) {
```

Comment:
For this to work properly (meaning that it is only applied when the saber is OFF) it has to be in an else section that matches up with the if on line 89.

Otherwise, this part runs both when the saber is on and when it's off, which makes it impossible to set the swing speed to something higher when the saber is on.

Reply:
that explains why some users couldn't adjust, did not notice that, fixed